gateway/src/node/client_handling/embedded_network_requester/mod.rs

```diff
@@ -28,6 +28,17 @@ impl LocalNetworkRequesterHandle {
         }
     }
 
+    // TODO: generalize this whole thing to be general. And change the name(s).
```

Comment:
just a comment so you wouldn't forget your own comment here : )

Reply:
yep, although that comment is for future PRs :)